risc0/zkvm/prove/method_id.h

```diff
@@ -14,28 +14,19 @@
 
 #pragma once
 
-#include <array>
-#include <memory>
+#include <cstddef>
+#include <cstdint>
+#include <string>
 
-#include "risc0/core/util.h"
-#include "risc0/zkp/core/sha256.h"
-#include "risc0/zkvm/circuit/constants.h"
+#include "risc0/zkvm/verify/riscv.h"
 
 namespace risc0 {
 
-static constexpr size_t kCodeDigestCount = log2Ceil(kMaxCycles / kMinCycles) + 1;
+constexpr size_t kDefaultCodeDigestCount = 12;
```

Comment:
Can we document what this does?  Would it make sense to have it in the same place as the kMaxCodeDigestCount?